### DISCUSSION TITLE
2021.02.26 금 회의록

### DISCUSSION THREAD

**Discussion by hakju** [hakju/ios-cloud-notes#28] [General]
불금이네요 🔥
1. TIL 꼭! 쓰기 (CoreData, Respoder 등)
2. core data 공부!
3. STEP2 준비?
4. 잠자기전에 TWL 작성하기

**Comment by soleJin** [comment]
+ STEP2 준비 (Core Data 공부 조금?!ㅋㅋㅋ)
+ TWL
